src/server/routes/application_settings.js

```diff
@@ -5,10 +5,9 @@ const express = require('express')
 const router = express.Router()
 const { applicationSettings } = require('../db/settings')
 
-router.get('/', function (req, res) {
-  applicationSettings().then(application_settings =>
-    res.json({ application_settings })
-  )
+router.get('/', async function (req, res) {
```

Comment:
Note that an error in this `async` function is technically unhandled (in a way that the new linter will not catch) since Express does not properly support async functions.

In my other PR #282 I ended up adding a third party module `express-async-errors` that addresses this as a sort of drive-by fix; though maybe that would fit better in this PR or separately-- it just happened to be useful while I was debugging my changes.

Reply:
It sounds like express 5.0 fixed this.  I wonder how big of an upgrade that is.

https://expressjs.com/en/guide/migrating-5.html#rejected-promises

Reply:
Express 5 is still listed as beta, so I don't think we should really consider switching to it yet.

Reply:
Ah, you're right.  Well, no rush.